packages/node_modules/@webex/webex-core/src/lib/services/services.js

```diff
@@ -278,7 +278,7 @@ const Services = WebexPlugin.extend({
    */
   collectPreauthCatalog({email} = {}) {
     if (!email) {
-      return Promise.reject(new Error('`email` is required'));
+      return this.updateServices({from: 'limited', query: {mode: 'DEFAULT_BY_PROXIMITY'}});
```

Comment:
if they don't provide an email, instead use the query param `mode: 'DEFAULT_BY_PROXIMITY'` to fetch based on proximity.

Reply:
Great implementation, if you could, please append an additional test to this describe method here: `packages/node_modules/@webex/webex-core/test/integration/spec/services/services.js L624` to assure that this is covered during any future change. You may need to modify the current tests as well.